pegasus/sites.v3/code.org/views/mobile_header_responsive.haml

```diff
@@ -1,7 +1,47 @@
--# an ultra-simple mobile header
+%link{:href=>"/css/mobile_header.css", :rel=>"stylesheet"}
+%script{src: "/js/mobile_header.js"}
 
 #pageheader
   .container_responsive
     #left
-      %a{:href=>"/"}
+      %a{:href=>'/'}
         %img#logo{:src=>'/images/logo.png'}
+
+    #nav
+      %ul.hide-responsive-menu
+        %li
+          %a{href:resolve_url('/student')} Learn
+        %li
+          %a{href:resolve_url('/educate')} Teach
+        %li
+          %a{href:resolve_url('/promote')} Stats
+        %li
+          %a{href:resolve_url('/help')} Help Us
+        %li
+          %a#about-more{href: "#"}
+            About
+            %b.caret
+        %span.about-nav.hide-about
+          %li
+            %a{:href => "/about"} About Us
+          %li
+            %a{:href => "/about/leadership"} Leadership
+          %li
+            %a{:href => "/about/donors"} Donors
+          %li
+            %a{:href => "/about/partners"} Partners
+          %li
+            %a{:href => "/about/team"} Full Team
+          %li
+            %a{:href => "/about/news"} Newsroom
+          %li
+            %a{:href => "/about/evaluation"} Evaluation
+          %li
+            %a{:href => "/about/jobs"} Jobs
+          %li
+            %a{:href => "/contact"} Contact Us
+          %li
+            %a{:href => "/faq"} FAQs
+
+      %a#nav-hamburger{href: "#"}
```

Comment:
I think this adds # to the URL.  Is there a way to avoid that?  (I think we can not use an <a> tag and just set the mouse pointer to make it look like it's one instead.)